avocado/utils/genio.py

```diff
@@ -201,3 +202,25 @@ def write_file_or_fail(filename, data):
     except OSError as details:
         raise GenIOError("The write to %s failed: %s" % (
                          filename, details))
+
+
+def find_pattern(filename, pattern):
+    """
+    Module for match pattern in a specified file
+
+    :param filename: Path to file
+    :type filename: str
+    :param pattern: pattern that need to match in file
+    :type pattern: str
+    returns True if the pattern is present in the file, False if not.
```

Comment:
This causes the travis failure. It's because you have block of sphinx notation (`:param ...:`, etc.) and then normal string without an empty line. Anyway proper fix is to use `:return: True if ...` as that is a valid sphinx tag.